terraform/helm/kube-state-metrics/values.yaml

```diff
@@ -0,0 +1,5 @@
+kube-state-metrics:
+  namespaceOverride: kube-system
```

Comment:
let's stop installing stuff into kube-system imo. We should just install these things into their own namespaces.

Reply:
Ok, I'll move the to another namespace, like "aptos-monitoring".

Reply:
Moved them to namespace "monitoring".